### PR TITLE
Schema testing

### PR DESCRIPTION
**Pull request by manuelq12**
New test added to test the structure of the response of a Github API request

### CODE REVIEW

File: test/GithubApi.Contract.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    let response;
+
+    before(async () => {
+      response = await agent
+        .get(`${urlBase}/events`)
+        .set('User-Agent', 'agent')
+        .auth('token', process.env.ACCESS_TOKEN);
+    });
+
+    it('then the body should have a schema', () => expect(response).to.be.jsonSchema(listPublicEventsSchema));
```
Comment (leonleo997) [inline test/GithubApi.Contract.test.js:21]: `Then the body should have the right schema`

**Review by leonleo997 — CHANGES_REQUESTED** [review]

File: test/GithubApi.Contract.test.js (hunk 1 above)
Comment (manuelq12) [reply test/GithubApi.Contract.test.js:21]: On my way

Comment (manuelq12) [reply test/GithubApi.Contract.test.js:21]: Done!

File: test/schema/ListPublicEvents.schema.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    },
+    body: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          id: {
+            type: 'string'
+          },
+          type: {
```
Comment (leonleo997) [inline test/schema/ListPublicEvents.schema.js:15]: how can you improve the verification for this property? is this property more than a string? how can you validate the specific set of values?

**Review by leonleo997 — CHANGES_REQUESTED** [review]

File: test/schema/ListPublicEvents.schema.js (hunk 2 above)
Comment (manuelq12) [reply test/schema/ListPublicEvents.schema.js:15]: Only for that property?

**Review by leonleo997 — APPROVED** [review]

**Pull request merged**